### PR TITLE
Fix/spark 440478 keep the breakout roster info during closing

### PR DESCRIPTION
**Pull request by othellashao**
# COMPLETES #
 https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-440478

## This pull request addresses

keep the breakout rosters info when breakouts is closing

## by making the following changes

1. Drop the update breakout sessions data into `breakouts.breakouts` since need to keep the breakout roster info during closing. Locus DTO will send the `undefined` or only `Main` sessions info here,  that will cause the breakout roster info is missed in this case since current breakout roster is bound to `breakouts.breakouts`
2. Clear the 'breakouts.breakouts' collection when current breakouts is not in progress

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by mickelr — APPROVED** [review]
LGTM

File: packages/@webex/plugin-meetings/src/breakouts/index.ts [hunk 1]
```diff
@@ -296,14 +296,30 @@ const Breakouts = WebexPlugin.extend({
     });
   },
 
+  /**
+   * get current breakout is in progress or not
+   * @returns {boolean}
+   */
+  getIsBreakoutInProgress() {
```
Comment (arun3528) [inline packages/@webex/plugin-meetings/src/breakouts/index.ts:303]: i would just change the name to isBreakoutInProgress 

we normally only add get when there is a API needed

**Review by arun3528 — APPROVED** [review]
just name change rest looks good

File: packages/@webex/plugin-meetings/src/breakouts/index.ts (hunk 1 above)
Comment (othellashao) [reply packages/@webex/plugin-meetings/src/breakouts/index.ts:303]: @arun3528 thanks, I will change it.

**Pull request merged**